scripts/bundleSizeConfig.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 629,
-  MAX_SIZE: 681,
+  MIN_SIZE: 634,
```

Comment:
This might cause the builds to fail - the smallest bundle is still probably 629K, but let's wait for builds to run first before we update this.